tox.ini

```diff
@@ -3,20 +3,20 @@ envlist = py27, py27-tornado23, py27-tornado31, docs, flake8
 
 [testenv]
 sitepackages = true
-commands = nosetests -v --with-xunit --xunit-file=xunit-{envname}.xml --with-coverage --cover-package=mopidy
+commands = py.test --junit-xml=xunit-{envname}.xml --cov=mopidy
```

Comment:
Confirmed.

I removed the inspect+warnings code, leaving the empty decorator in place as a place to add e.g. logging when cleaning up code using deprecated properties.